DigitalLearningSolutions.Web/Views/SuperAdmin/Faqs/_FaqCard.cshtml

```diff
@@ -0,0 +1,61 @@
+<div class="searchable-element nhsuk-expander-group word-break" id="@Model.FaqId-card">
+
+  <details class="nhsuk-details nhsuk-expander">
+    <div class="nhsuk-details__text">
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-one-quarter">
+          <div class="nhsuk-tag tag card-tag--grey">
+            @Model.TargetGroup
```

Comment:
Again, we are using Models here without specifying the model type. In this case, I think it would definitely be useful to specify the view model.